e2etest/newe2e_task_runazcopy.go

```diff
@@ -337,8 +342,10 @@ func RunAzCopy(a ScenarioAsserter, commandSpec AzCopyCommand) (AzCopyStdout, *Az
 		0, command.ProcessState.ExitCode())
 
 	a.Cleanup(func(a ScenarioAsserter) {
-		UploadLogs(a, out, stderr, DerefOrZero(commandSpec.Environment.LogLocation))
-		_ = os.RemoveAll(DerefOrZero(commandSpec.Environment.LogLocation))
+		if !commandSpec.Environment.ManualLogin {
```

Comment:
This also interests me-- I suspect this is to prevent capturing the login token, but, could we not filter that out?

Reply:
Currently the new e2e test framework is designed such that each of the scenarios creates a new temp log location. When you log in it creates a file in the log location and we want that log in to be the same across all e2e test framework logs, so we avoid removing the log folder. There is probably  a cleaner way to do this that we can look into during a cleanup PR.